### DISCUSSION TITLE
log permission

### DISCUSSION THREAD

**Discussion by alicera** [megahertz/electron-log#329] [General]
user 'a' generate the log/main.log
user 'b' want to write the log/main.log

I meet the permission.
How can I fix it on the code not the command chmod on terminal.
I want the user 'b' can directly use the log/main.log without dothing.
 

https://github.com/megahertz/electron-log#options-1

**Comment by megahertz** [comment]
Normally, each user has its own log directory. When your app writes the same file for different users you should control file permissions by yourself. The easiest way it to set log file permissions to 0666 on app start.

**Comment by alicera** [comment]
Here is the right code?
```
process.umask(0666)
```

**Reply by megahertz** [reply]
fs.chmod

**Comment by alicera** [comment]
ok, thanks.

I process the permission with

```
touch /opt/app/main.log # I dont know when it generate with electron-log
 , so create it.
chmod g+w  /opt/app/main.log
```
